test/unit/test-amp-story-player.js

```diff
@@ -640,5 +640,23 @@ describes.realWin('AmpStoryPlayer', {amp: false}, (env) => {
         remaining: 4,
       });
     });
+
+    it('signals when player changed story using go method', async () => {
+      const playerEl = win.document.createElement('amp-story-player');
+      appendStoriesToPlayer(playerEl, 5);
+
+      const player = new AmpStoryPlayer(win, playerEl);
+
+      await player.load();
+
+      const navigationSpy = env.sandbox.spy();
+      playerEl.addEventListener('navigation', navigationSpy);
+      player.go(1);
+      expect(navigationSpy.firstCall.args[0].type).to.eql('navigation');
+      expect(navigationSpy.firstCall.args[0].detail).to.eql({
+        index: 1,
+        remaining: 3,
+      });
```

Comment:
It makes no difference at all, but if you're interested there's a helper to write this: `expect(navigationSpy).to.have.been.calledOnceWithExactly(params)`